.github/workflows/build_artifacts.yml

```diff
@@ -47,6 +47,15 @@ jobs:
             CPU=aarch64 make
             bazel shutdown
 
+      - name: Test example
+        run: |
+            cd example
+            mkdir -p build
+            cd build
+            cmake ..
```

Comment:
Should we be building with bazel here?

Reply:
Oh the example was deliberately written in cmake so as to show people they can just download the binaries from github actions and use them without bazel